From 884f1cfe9ec745101e4542583f84c890c2e9ef65 Mon Sep 17 00:00:00 2001
From: Cyril Rohr <hey@cyrilrohr.com>
Date: Tue, 29 Sep 2026 13:55:23 +0200
Subject: [PATCH 8/8] go.mod: require golang-jwt directly

The gha compression test builds a runtime token with golang-jwt, so the
module is now a direct dependency. go mod tidy moves it out of the
indirect block; go.sum and vendor/ are unchanged.

Signed-off-by: Cyril Rohr <hey@cyrilrohr.com>
---
 go.mod | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/go.mod b/go.mod
index 2f1b2989c..953aebcf9 100644
--- a/go.mod
+++ b/go.mod
@@ -40,6 +40,7 @@ require (
 	github.com/docker/go-units v0.5.0
 	github.com/gofrs/flock v0.13.0
 	github.com/gohugoio/hashstructure v0.6.0
+	github.com/golang-jwt/jwt/v5 v5.3.1
 	github.com/google/go-cmp v0.7.0
 	github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510
 	github.com/hashicorp/go-cleanhttp v0.5.2
@@ -181,7 +182,6 @@ require (
 	github.com/go-openapi/swag/yamlutils v0.26.1 // indirect
 	github.com/go-openapi/validate v0.26.0 // indirect
 	github.com/go-viper/mapstructure/v2 v2.5.0 // indirect
-	github.com/golang-jwt/jwt/v5 v5.3.1 // indirect
 	github.com/golang/groupcache v0.0.0-20241129210726-2c02b8208cf8 // indirect
 	github.com/google/certificate-transparency-go v1.3.3 // indirect
 	github.com/google/go-containerregistry v0.21.7 // indirect
-- 
2.51.0

